src/index.ts

```diff
@@ -172,14 +179,18 @@ export class I18N {
                 result = result.replace(new RegExp(`({{${param}}})`, 'g'), replacer)
             });
         } else {
-            result = keyValue;
+            result = keyValue as string;
+
+            if (Array.isArray(result)) {
+                this.warn('Using plural key without params', keysetName, key);
+            }
```

Comment:
Как-то странно возвращать массив когда по типам должна быть строка 
```suggestion
            if (Array.isArray(result)) {
                this.warn('Using plural key without params', keysetName, key);
                return key;
            }
            
            result = keyValue;
```

Reply:
Может лучше переделать как-то так?:
```ts
if (Array.isArray(keyValue)) {
  const count = Number(params?.count);
  if (isNaN(count)) {
      this.warn('Missing params.count for key.', keysetName, key);
      return key;
  }
   
   ...
} else {
   result = keyValue;
}

if (params) {
    Object.keys(params).forEach((param) => {
      ...
    });
}
```

Reply:
Распрямил немного условия